packages/react/src/themes/teams/components/Table/tableVariables.ts

```diff
@@ -0,0 +1,55 @@
+import { pxToRem } from '../../../../lib'
+
+export interface TeamsTableVariables {
+  viewMode: 'default' | 'compact'
+  color: string
+  hoverColor: string
+  backgroundColor: string
+  backgroundHoverColor: string
+  borderWidth: string
+  headerBorderColor: string
+  headerBorderHoverColor: string
+  headerBorderFocusColor: string
+  rowBorderColor: string
+  rowBorderHoverColor: string
+  rowBorderFocusColor: string
+  cellBorderColor: string
+  cellBorderHoverColor: string
+  cellBorderFocusColor: string
+  defaultRowHeight: string
+  compactRowHeight: string
+  minCellWidth: string
+  cellPadding: string
+  rowPadding: string
+  headerFontSize: string
+  bodyFontSize: string
+  cellContentOverflow: 'none' | 'ellipsis'
+}
+
+export default (siteVariables): Partial<TeamsTableVariables> => {
+  return {
+    viewMode: 'default',
+    color: siteVariables.colorScheme.default.foreground,
+    hoverColor: siteVariables.colorScheme.default.foreground,
```

Comment:
`hoverColor: siteVariables.colorScheme.default.foreground,` should be `hoverColor: siteVariables.colorScheme.default.foregroundHover`